LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/BasicsStation/DownlinkMessage.cs

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoRaTools.LoRaPhysical
```

Comment:
Add nullability checks for new code:

```suggestion
#nullable enable

namespace LoRaTools.LoRaPhysical
```